packages/nextjs/utils/toast.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { CheckmarkIcon, ErrorIcon, toast as hotToast } from "react-hot-toast";
+import { XMarkIcon } from "@heroicons/react/20/solid";
+import { ExclamationTriangleIcon, InformationCircleIcon } from "@heroicons/react/24/solid";
+import Spinner from "../components/Spinner";
+
+type TToastProps = {
+  message: React.ReactNode;
+  status: "success" | "info" | "loading" | "error" | "warning";
+  duration?: number;
+  icon?: string;
+};
+
+type ToastOptions = {
+  duration?: number;
+  icon?: string;
+};
+
+const hotToastIconStyles: React.CSSProperties = {
+  backgroundColor: "#7eaaef",
+  marginTop: "0.25rem",
+};
+
+const ENUM_STATUSES = {
+  success: <CheckmarkIcon style={hotToastIconStyles} />,
+  loading: <Spinner />,
+  error: <ErrorIcon style={hotToastIconStyles} />,
+  info: <InformationCircleIcon className="w-7  text-primary-focus" />,
+  warning: <ExclamationTriangleIcon className="w-7 text-primary-focus" />,
+};
+
+const DEFAULT_DURATION = 3000;
+
+/**
+ * Custom Toast
+ */
+function Toast({ message, status, duration = DEFAULT_DURATION, icon }: TToastProps) {
```

Comment:
Created it inside `packages/nextjs/utils`, please let me know if we should move it to `packages/nextjs/components`. 

OR 

we can only move this `Toast` jsx component to `packages/nextjs/components`  

also please let me if the current structure and how the toast are called is good or not, happy to revamp and give it an another try 🙌

Reply:
Utils is fine for me. Inside `utils/scaffold-eth` might be better tho.